tests/deepsparse/transformers/pipelines/test_text_generation.py

```diff
@@ -29,7 +29,7 @@ def pipeline():
     return Pipeline.create(
         task="text_generation",
         model_path="hf:mgoin/TinyStories-1M-deepsparse",
-        engine_type="onnxruntime",
```

Comment:
in general using onnxruntime is preferred in tests, since it is faster to compile.

Reply:
ah I changed it locally and forgot to change it back. Good catch!